aiohttp/web.py

```diff
@@ -287,3 +287,38 @@ def __call__(self):
 
     def __repr__(self):
         return "<Application>"
+
+
+def run_app(app, *, host='0.0.0.0', port=None, loop=None,
+            shutdown_timeout=60.0, ssl_context=None):
+    """Run an app locally"""
+    if port is None:
+        if not ssl_context:
+            port = 8080
+        else:
+            port = 8443
+
+    if loop is None:
```

Comment:
Is an explicit `loop` necessary -- Wouldn't `loop = app.loop` suffice here?
Is there a use case where the App would run on a different loop than the Server?